meetings/2018-06-13.md

```diff
@@ -0,0 +1,111 @@
+
+* Nominating bcoe as a collaborator [#20242](https://github.com/nodejs/node/issues/20242)
+  * Seems like everybody on the issue is +1, any objections.  Myles will find somebody to
+    Onboard.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * Nothing to add since last board meeting
+  * Work on getting CommComm charter in place for Director etc.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Modules
+    * Have gotten through a larger number of the use cases
+    * PR open for meta require
+    * Starting to talk about deadlines
+    * Main contentious topics are about transparent interop, module specifier resolution
+    * Package name maps proposal from Domenic, to allow install time tooling to generate
+      manifest to be consumed by browser to allow bare imports.  May provide path to being
+      to use modules in both browsers and Node.js without changes. Does not affect CJS
+
+  * N-API
+     * Continued push on doc for node-addon-api
+     * Looks like we’ll have workshop and a few talks at Node Summit
+     * Proposal for adding new APIs as experimental
+     * Proposal for guidance on adding new APIs
+
+  * OpenSSL Evolution
+    * Pull request 21282, would like some additional opinions from TSC members. Relates to security issue,
+      CVE is assigned and details are public but not in OpenSSL release yet as its low severity.
+    * Anna, would this be considered a breaking change if not a security issue?
+
+  * Workers
+    * Not too much new since PR landed, may include in next 10.x release (it is behind a flag)
+    * working out minor issues.
+
+  * TSC Governance
+    * Some discussion on code owners, mixed feelings
+    * Discussion around backports, and our definition of done.  Should we not close
+      pull requests until all backports audited, landed etc.
+
+  * New Streams APIs
+    * Progress at collab summit, Jerimiah, fs-source and fs-sync modules, more status
+      In fishrock123/bob repo.   Nothing new since collab summit.
+
+* Proposal: add all new core modules under a scope? (too late for http2) [#389](https://github.com/nodejs/TSC/issues/389)
+  * Myles, we need namespaces and we need to move on faster than we have to. Every
+    time we add something new we have the shadowing issue with npm
+  * We have a number of PRs open (mime, brotili) and other recent ones have had
+    collisions.  Also was issue for fs.promises.
+  * Questions about what kind of symbol we use, Myles highly recommends @nodejs
+    namespace
+  * TC39 is also looking into namespaces as a language feature, advice from committee was
+    not to wait for it (there are no proposals now, likely more than a year out).
+  * Can’t just be a directory, security concern about versions that don’t have the namespace as
+    people coule publish modules with the same name in those versions and they could be used
```

Comment:
This is not, in fact, any more of a concern than new bare core modules - a namespace on npm is a *user* - and node would own whatever user it was, so it would be impossible for anyone to publish a module under that namespace.